### ISSUE TITLE
Changed SOC parameter values for Summer and Fall 2018

### ISSUE THREAD

**Issue by wuhugh** [Rolstenhouse/uf_api#3]
The endpoints for the schedule of courses are actually a bit off for Fall 2018.

In the summer the base URL is actually `https://one.uf.edu/apix/soc-summer/schedule/` (this is actually the base URL for Spring too for some reason0. The `term` you can append the letters A, B, or C for Summer A, B, or C.

The `term` for Fall is `2188` and the `program` isn't `RES` for regular, but `CWSP` instead.
`https://one.ufl.edu/apix/soc/schedule/?category=CWSP&term=2188`

**Comment by Rolstenhouse** [comment]
Awesome catch! I think UF is  in the process of updating their API. I'll do a little more investigation and see if I can figure out everything else that's changed.

**Comment by Luc-Olsthoorn** [comment]
Yeah they seem to be changing quite a bit in naming conventions. You can see some of the changes of API between https://one.uf.edu/apix/soc-summer/filters and https://one.uf.edu/apix/soc/filters

**Comment by Luc-Olsthoorn** [comment]
Some more things that have changed for fall:
1. The web courses used to have specific codes in the JSON("P", "H", "C", "A"); now they are all "P"
2. The LASTROWS is now last-control-number

**Comment by wuhugh** [comment]
### y tho

**Comment by Rolstenhouse** [comment]
I think I made some changes to reflect it. Do you guys think I should mark that API as unstable?

**Comment by wuhugh** [comment]
I think it's fine (maybe). I don't think they'll change the current endpoints now.

**Comment by Luc-Olsthoorn** [comment]
https://one.uf.edu/api/myschedule/course-search/
This is another endpoint that needs the login, but it returns a similar array, but with the number of seats available there.

**Comment by wuhugh** [comment]
What do we pass the login as?

**Comment by Rolstenhouse** [comment]
I think you already have to be logged in right @Luc-Olsthoorn ?
